swarm/pss/pss.go

```diff
@@ -150,14 +186,12 @@ func (self *Pss) APIs() []rpc.API {
 			Service:   NewAPI(self),
 			Public:    true,
 		},
-	}
-	if self.debug {
-		apis = append(apis, rpc.API{
-			Namespace: "pss",
+		rpc.API{
+			Namespace: "psstest",
 			Version:   "0.1",
 			Service:   NewAPITest(self),
 			Public:    true,
```

Comment:
All APIs are exposed via IPC regardless of this flag, with only public APIs being exposed via HTTP / WebSockets (although that can be overridden wit the `--rpcapi` and `--wsapi` flags).